test/test_local.py

```diff
@@ -200,6 +200,37 @@ def test_save(self):
         self.remote_node.pdo.save()
         self.local_node.pdo.save()
 
+    def test_send_pdo_on_operational(self):
+        self.local_node.tpdo[1].period = 0.5
+
+        self.local_node.nmt.state = 'INITIALISING'
+        self.local_node.nmt.state = 'PRE-OPERATIONAL'
+        self.local_node.nmt.state = 'OPERATIONAL'
+
+        self.assertNotEqual(self.local_node.tpdo[1]._task, None)
+
+    def test_config_pdo(self):
+        # Disable tpdo 1
+        self.local_node.tpdo[1].enabled = False
+        self.local_node.tpdo[1].cob_id = 0
+        self.local_node.tpdo[1].period = 0.5 # manually assign a period
```

Comment:
I don't understand that comment.  Just above, you do the same without a comment.  Is this intended to test mix & match between direct assignment and SDO based changes?

Isn't it using the value from the inhibit time object anyway?

Reply:
It doesn't use inhibit time currently. I have a different branch for this, I wasn't sure if I should try to pull it in with this stuff.  https://github.com/grantweiss-RaymondCorp/canopen/commit/0a9f50da74e1e3ed3c9a18a13e747a6a1a484772

Reply:
The approach of falling back to `event_timer` looks reasonable.  But I agree we don't have to do it all in this PR, but can iterate with more improvements / features once this is merged.